TestAssets/TestProjects/TestAppSimple/project.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.0-rc2-*"
```

Comment:
Good point, I will add rids.

I don't want to do type: platform only because I think there's cases where we want a runnable build output (without dotnet run). Feel free to disagree though.